[UPDATE BONUS GUIDE] Sparrow Terminal: Add mixing section (#1207)
raspibolt/raspibolt: guide/bonus/bitcoin/sparrow-terminal.md

  ![Sparrow_Test](../../../images/sparrow-test.png)

### Mix Bitcoin with Sparrow Terminal

* Install tmux

  ```sh
  $ sudo apt update

>   $ sudo apt full-upgrade

Reviewer: ```suggestion
```
This line is not necessary, only line before updating and refresh new releases packages

>   $ sudo apt install tmux

Reviewer: Why install tmux? why is useful in this case of use?

Author: > Why install tmux? why is useful in this case of use?

AFAIK to run sparrow mixing, wallet needs to stay open in order to work. ST mixing client cannot be run as a service yet - wallet needs to be opened on a boot -> unencrypted + if passphrase present, confirmed by user by typing. Then wallet is kept open and mixing happens. 

Using ctrl + C wallet is closed, so installing TMUX and keeping wallet open on a separate session is necessary. If there is any way without installing tmux / additional software and run sparrow on a background, then that could be preferably used instead of tmux

Reviewer: Screen could be the solution --> https://linuxize.com/post/how-to-use-linux-screen/

Author: looks great, will test

Author: > Screen could be the solution --> https://linuxize.com/post/how-to-use-linux-screen/

screen does not show the ST GUI properly, the screen is broken. Trying to find solution but no success yet

Reviewer: I agree! Tmux worked well for me. We will work to release the mixing section!

  ```

* Start a new tmux session called "Sparrow"

  ```sh

>   $ tmux new -sSparrow

Reviewer: missing space? change to:
```suggestion
  $ tmux new -s sparrow_server
```

[Source](https://www.howtogeek.com/671422/how-to-use-tmux-on-linux-and-why-its-better-than-screen/)
**(Starting a Named tmux Session)** section

  ```

* Launch Sparrow Terminal

  ```sh
  $ Sparrow
  ```

> * Connect Sparrow Terminal to your own Electrs implementation according to steps above 

Reviewer: Change to-> Connect Sparrow Terminal to your own Electrum server implementation according to steps above

> * Connect Sparrow Terminal to your own Electrs implementation according to steps above 

Reviewer: ```suggestion
* Connect Sparrow Terminal to your own Electrum Server implementation according to the steps above 
```
"To include Fulcrum"

* Enter SCODE if available, you will get discount on pool fee. You can monitor SCODEs by following Samourai Wallets twitter account, Whirlpool or SW Telegram group etc.

* Mix selected funds 

* Once confirmed, go to `Accounts > Postmix > UTXOs > Mix To`.

* You can mix to cold storage if desired. Select value for minimum mixes before sending to cold storage

* If you use Whirlpool with Dojo as well - set Postmix index range to "odd". This way you improve chances of getting into a mix by running two separate mixing clients at the same time, using different index ranges to prevent mix failures

* Detach tmux session to run ST on the background: 
1. press `ctrl + b` once 
2. press `d` once

Closing or logging out from your node without detaching would cause mixing to stop. ST now runs as a separate process regardless of you disconnecting from node

* You can view tmux sessions using following command

  ```sh
  $ tmux ls
  ```

* You can get back in sessions using

  ```sh
  $ tmux attach
  ```

> 

Reviewer: ```suggestion
or use this if you have other sessions opened
  $ tmux a -t sparrow_server
```
